variants/meshlink/variant.h

```diff
@@ -0,0 +1,233 @@
+/*
+  Copyright (c) 2014-2015 Arduino LLC.  All right reserved.
+  Copyright (c) 2016 Sandeep Mistry All right reserved.
+  Copyright (c) 2018, Adafruit Industries (adafruit.com)
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#ifndef _VARIANT_MESHLINK_
+#define _VARIANT_MESHLINK_
+#ifndef MESHLINK
+#define MESHLINK
+#endif
+/** Master clock frequency */
+#define VARIANT_MCK (64000000ul)
+
+//#define USE_LFXO // Board uses 32khz crystal for LF
+ #define USE_LFRC    // Board uses RC for LF
+
+/*----------------------------------------------------------------------------
+ *        Headers
+ *----------------------------------------------------------------------------*/
+
+#include "WVariant.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+
+// Number of pins defined in PinDescription array
+#define PINS_COUNT (48)
+#define NUM_DIGITAL_PINS (48)
+#define NUM_ANALOG_INPUTS (2)
+#define NUM_ANALOG_OUTPUTS (0)
+
+
+
+#define BUTTON_PIN (38) // If defined, this will be used for user button presses,
+#define BUTTON_NEED_PULLUP
+
+
+// LEDs 
+#define PIN_LED1 (24) //24
+#define PIN_LED2 (-1)  //Watchdog enable pin25
+//#define PIN_LED3 (-1)
+
+//#define LED_RED PIN_LED1
+//#define LED_GREEN LED_PIN
+#define LED_BLUE PIN_LED1
+
+//#define LED_CONN PIN_LED1
+#define LED_BUILTIN PIN_LED1
+
+#define LED_STATE_ON 0 // State when LED is litted
+#define LED_INVERTED 1
+// Testing USB detection
+//#define NRF_APM
+/*
+ * Analog pins
+ */
+#define PIN_A0 (2)
+#define PIN_A7 (31)
+/*
+static const uint8_t A0 = PIN_A0;
```

Comment:
Please remove the RAK4631 specific comments. These will confuse developers working on this define.